themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/layout.html

```diff
@@ -209,11 +209,15 @@ <h3>News</h3>
             {% endif %}
             {% endif %}
 
-            {% if request.is_sidebar? %}
-            {% include "columns_2" %}
-            {% else %}
+            {% comment %}
+              {% if request.is_sidebar? %}
+              {% include "columns_2" %}
+              {% else %}
+              {% include "columns_1" %}
+              {% endif %}
```

Comment:
Nit: Does this need to stay? Same question for the newly commented out lines in pages_show_volunteer_signup.html and pages_show_volunteer_signup_wide.html